src/Appwrite/Utopia/Response/Model/File.php

```diff
@@ -13,37 +13,44 @@ public function __construct()
             ->addRule('$id', [
                 'type' => self::TYPE_STRING,
                 'description' => 'File ID.',
+                'default' => '',
                 'example' => '5e5ea5c16897e',
             ])
             ->addRule('$permissions', [
                 'type' => Response::MODEL_PERMISSIONS,
                 'description' => 'File permissions.',
+                'default' => '',
```

Comment:
Can this also be `new \stdClass` on default?

Reply:
Fixed :)